### PR TITLE
Added more functionality for Limitlessled (RGBWW, RGBW) v6 and added direct disco mode and nightlight for v4/5

### PR DESCRIPTION
**Pull request by chrisverhoef**

### CODE REVIEW

**Comment by gizmocuz** [comment]
If possible, could you make the commands/status match ? For example

case Limitless_DiscoM1:
lstatus="Set to Disco 1";
break;

if (switchcmd == "Disco Mode 1")
cmd = Limitless_DiscoM1

you see the above don't match, i would change it to "Disco Mode 1"

File: main/RFXNames.cpp [hunk 1]
```diff
@@ -1478,32 +1478,32 @@ void GetLightStatus(
 		case Limitless_NightMode:
 			lstatus="Set to Night";
 			break;
-		case Limitless_DiscoM1:
-			lstatus="Set to Disco 1";
+		case Limitless_DiscoMode_1:
+			lstatus="Set to Disco Mode 1";
```
Comment (gizmocuz) [inline main/RFXNames.cpp]: could you remove the prefix 'Set to '?

Comment (chrisverhoef) [reply main/RFXNames.cpp]: I will do.

**Comment by gizmocuz** [comment]
Thanks!

**Comment by chrisverhoef** [comment]
And thanks to deennoo for testing!

**Pull request merged**